File: Java/SpaceVIL/spacevil/src/main/java/com/spacevil/Frame.java
package com.spacevil;

public class Frame extends Prototype {
    private static int count = 0;

    /**
     * Constructs a Frame
     */
    public Frame() {
        setItemName("Frame_" + count);
        count++;
        // setStyle(DefaultsService.getDefaultStyle("SpaceVIL.Frame"));
        setStyle(com.spacevil.Common.DefaultsService.getDefaultStyle(Frame.class));
        isFocusable = false;
    }

    @Override
    public boolean getHoverVerification(float xpos, float ypos) {
        return false;
    }
}

File: spacevil_21052019/net_src/Items/Frame.cs
using System.Drawing;
using SpaceVIL.Common;

namespace SpaceVIL
{
    public class Frame : Prototype
    {
        static int count = 0;

        /// <summary>
        /// Constructs a Frame
        /// </summary>
        public Frame()
        {
            SetItemName("Frame_" + count);
            count++;

            SetStyle(DefaultsService.GetDefaultStyle(typeof(SpaceVIL.Frame)));
            IsFocusable = false;
        }

        protected internal override bool GetHoverVerification(float xpos, float ypos)
        {
            return false;
        }
    }
}
